tardis/spectrum/formal_integral/formal_integral_solver.py

```diff
@@ -0,0 +1,65 @@
+
+from tardis.spectrum.base import TARDISSpectrum
+from tardis.spectrum.formal_integral.source_function import SourceFunctionSolver
+
+
+def FormalIntegralSolver():
```

Comment:
Does this need to exist in this PR? If it's non-functional, I would suggest adding `pass` or raise a `NotImplemented` error.